corehq/apps/style/templates/style/includes/global_navigation_bar.html

```diff
@@ -4,6 +4,35 @@
 {% load tzmigration %}
 
 {% if request.couch_user %}
+    <script type="text/javascript">
+        $('#settingsmenu-projectSettings').click(function () {
+            ga_track_event('Nav Bar', 'Click Gear Icon');
+        });
```

Comment:
I think these should all be `gaTrackLink()` calls, which will ensure that the call to the google analytics completes before we navigate away from the current page.